dotnetv3/Glue/Scenarios/Glue_Basics/GlueBasics.cs

```diff
@@ -0,0 +1,175 @@
+            }
+            while (crawlerState != "READY");
+            Console.WriteLine($"The crawler {crawlerName} is now ready for use.");
+        }
+        else
+        {
+            Console.WriteLine($"Couldn't create crawler {crawlerName}.");
+            return; // Exit the application.
+        }
+
+        uiWrapper.DisplayTitle("Start AWS Glue crawler");
+        Console.WriteLine("Now let's wait until the crawler has successfully started.");
+        var crawlerStarted = await wrapper.StartCrawlerAsync(crawlerName);
+        if (crawlerStarted)
+        {
+            CrawlerState crawlerState;
+            do
+            {
+                crawlerState = await wrapper.GetCrawlerStateAsync(crawlerName);
+            }
+            while (crawlerState != "READY");
+            Console.WriteLine($"The crawler {crawlerName} is now ready for use.");
+        }
+        else
+        {
+            Console.WriteLine($"Couldn't start the crawler {crawlerName}.");
+            return; // Exit the application.
+        }
+
+        uiWrapper.PressEnter();
+
+        Console.WriteLine($"\nLet's take a look at the database: {dbName}");
+        var database = await wrapper.GetDatabaseAsync(dbName);
+
+        if (database != null)
+        {
+            uiWrapper.DisplayTitle($"{database.Name} Details");
+            Console.WriteLine($"{database.Name} created on {database.CreateTime}");
+            Console.WriteLine(database.Description);
+        }
+
+        uiWrapper.PressEnter();
+
+        var tables = await wrapper.GetTablesAsync(dbName);
+        if (tables.Count > 0)
+        {
+            tables.ForEach(table =>
+            {
+                Console.WriteLine($"{table.Name}\tCreated: {table.CreateTime}\tUpdated: {table.UpdateTime}");
+            });
+        }
+
+        uiWrapper.PressEnter();
+
+        uiWrapper.DisplayTitle("Create AWS Glue job");
+        Console.WriteLine("Creating a new AWS Glue job.");
+        var description = "An AWS Glue job created using the AWS SDK for .NET";
+        var success = await wrapper.CreateJobAsync(jobName, roleName, description, scriptUrl);
+
+        uiWrapper.PressEnter();
+
+        uiWrapper.DisplayTitle("Starting AWS Glue job");
+        Console.WriteLine("Starting the new AWS Glue job...");
+        var jobRunId = await wrapper.StartJobRunAsync(jobName);
+        var jobRunComplete = false;
+        do
+        {
+            var jobRun = await wrapper.GetJobRunAsync(jobName, jobRunId);
+            if (jobRun.JobRunState == "SUCCEEDED" || jobRun.JobRunState == "STOPPED" ||
+                jobRun.JobRunState == "FAILED" || jobRun.JobRunState == "TIMEOUT")
+            {
+                jobRunComplete = true;
+            }
+        } while (!jobRunComplete);
+
+        uiWrapper.DisplayTitle("Get AWS Glue job run information");
+        Console.WriteLine("Getting information about the AWS Glue job.");
+        var jobRuns = await wrapper.GetJobRunsAsync(jobName);
```

Comment:
I don't see steps 7 or 8 from the scenario spec here, am I overlooking them?

Reply:
Skipped unintentionally. I've added them.